### DISCUSSION TITLE
MIDI.sendNoteOff not working

### DISCUSSION THREAD

**Discussion by Olivierdemoustier** [FortySevenEffects/arduino_midi_library#229] [General]
It looks like MIDI.sendNoteOff is not doing anything.
This simple sketch will only send NoteOn. I sniff the midi with midi-ox

#include <MIDI.h>
MIDI_CREATE_DEFAULT_INSTANCE();

void setup() {
  MIDI.begin();  
}

void loop() {
  MIDI.sendNoteOn (55, 10, 11);
  delay(400);
  MIDI.sendNoteOff (55, 20, 11);
}

**Comment by Olivierdemoustier** [comment]
Stupid me. I had a filter in midi-OX that filters ou all NOTE-OFF's.
Really sorry

**Reply by franky47** [reply]
No worries, it happens!
